$NetBSD: patch-ae,v 1.3 2015/12/29 23:34:48 dholland Exp $

Use srand48() instead of srand().
XXX: should use random() or arc4random() or something else more suitable.

--- src/ick-wrap.c.orig	2010-09-01 22:21:34.000000000 +0000
+++ src/ick-wrap.c
@@ -59,7 +59,7 @@ $N
     globalargc = argc;
 #endif
     /* set seed for random error generation */
-#ifdef USG
+#if 1
     srand48(time(0) + getpid());
 #else
     srand(time(0));
